index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+
+    <title></title>
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <!--script src='main.js'></script-->
+</head>
+<body>
+    <header class="toolbar">
+        <img id="img1" src="image_geometry_1.svg" alt="">
+        <div class="ic_icon">
+          <button id="btn"></button>
+          <button id="btn"></button>
+          <button id="btn"></button>
+        </div>
+        
+    </header>
+    <section class="headline">
+        <div class="group48">
+            <p id="title">CharlieGiorgania</p>
```

Comment:
- [ ] Kindly wrap the main title with an `h1` tag. That will help the search engines to know about a specific topic, in this case, you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.